extensions/console-notification.crd.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: consolenotifications.config.openshift.io
+  annotations:
+    displayName: ConsoleNotification
+    description: Extension for configuring openshift web console notifications.
+spec:
+  group: config.openshift.io
+  version: v1
+  versions:
+  - name: v1
+    served: true
+    storage: true
+  scope: Cluster
+  names:
+    plural: consolenotifications
+    singular: consolenotification
+    kind: ConsoleNotification
+    listKind: ConsoleNotificationList
+  additionalPrinterColumns:
+  - name: Age
+    type: date
+    JSONPath: .metadata.creationTimestamp
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          type: object
+          description: Represents console notification spec
+          required:
+          - notification
+          - scope
+          properties:
+            notification:
+              type: object
+              description: Object that holds notification extension
+              required:
+                - text
+              properties:
+                text:
+                  type: string
+                  description: Text of the notification
+                link:
+                  type: object
+                  description: Object that holds notification link details
+                  required:
+                    - href
+                  properties:
+                    href:
```

Comment:
Any desire to validate the href?

Reply:
We could, but both relative and absolute links will work, so I'm not sure its worth it?

Reply:
Any intention to distinguish between internal and external links ?